src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -154,6 +155,12 @@ function WorkspaceNewRoomPage(props) {
         [translate],
     );
 
+    // the delay to be used for delayed focus on the room name input field
+    const inputFocusDelay = 600;
+
+    const roomNameInputRef = useRef(null);
+    useDelayedInputFocus(roomNameInputRef, inputFocusDelay);
```

Comment:
@akinwale 
Sorry for minor comment at last moment. Missed this, we actually don't need variable here. we can just put 600 here.


And do a comment on code, why we choose 600 as the value.

Reply:
@abdulrahuman5196 Done.